src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private static final Color[] COLORS = Color.values();
+    private static final Random random = new Random();
```

Comment:
make Random and Color not static. And I think you not need variable for list of enum)